website/docs/r/dax_cluster.html.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: "aws"
+page_title: "AWS: aws_dax_cluster"
+sidebar_current: "docs-aws-resource-dax-cluster"
+description: |-
+  Provides an DAX Cluster resource.
+---
+
+# aws\_dax\_cluster
+
+Provides a DAX Cluster resource.
+
+## Example Usage
+
+```hcl
+resource "aws_dax_cluster" "bar" {
+  cluster_id         = "cluster-example"
+  iam_role_arn       = "arn:aws:iams:us-east-1:012345678999"
```

Comment:
Especially since this is an invalid ARN, you can point at a fake data source like `${data.aws_iam_role.example.arn}`